rasterio/_warp.pyx

```diff
@@ -801,7 +801,7 @@ cdef class WarpedVRTReaderBase(DatasetReaderBase):
                  src_nodata=DEFAULT_NODATA_FLAG, nodata=DEFAULT_NODATA_FLAG,
                  width=None, height=None,
                  src_transform=None, transform=None,
-                 init_dest_nodata=True, src_alpha=0, add_alpha=False,
+                 init_dest_nodata=True, src_alpha=0, dst_alpha=0, add_alpha=False,
```

Comment:
```suggestion
                 init_dest_nodata=True, src_alpha=0, int dst_alpha=0, add_alpha=False,
```

Reply:
@snowman2 should I add this for all the other variables?